### ISSUE TITLE
[Bug] Assign object of type "Action" to property of type "Action_QMLTYPE_118*"

### ISSUE THREAD

**Issue by dgalli1** [korapp/plasma-homeassistant#51]
**Describe the bug**
After installing the extension it fails to load.

**Screenshots**

**Desktop (please complete the following information):**
 - Distribution: Tested on Manjaro and latest Arch
 - KDE Plasma 6.0.5
 - Frameworks 6.3.0

**Additional context**
file:///home/damian/.local/share/plasma/plasmoids/com.github.korapp.homeassistant/contents/ui/main.qml:13:25: Type FullRepresentation unavailable

file:///home/damian/.local/share/plasma/plasmoids/com.github.korapp.homeassistant/contents/ui/FullRepresentation.qml:66:28: Cannot assign object of type "Action" to property of type "Action_QMLTYPE_118*" as the former is neither the same as the latter nor a sub-class of it.

After removing the HelpfulAction i was able to load the plasmoid.
https://github.com/korapp/plasma-homeassistant/blob/0e43dfbb5ff80ca1e24a70d53b4d7025ae643dd2/package/contents/ui/FullRepresentation.qml#L66

**Comment by korapp** [comment]
This has already been investigated in #35
**TL;DR**
If you don't want to try deleting the QML cache or editing the plasmoid code you'll have to wait for Frameworks 6.4.0 to be available for your distribution.

**Comment by korapp** [comment]
Duplicate of #35
